aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -182,6 +207,12 @@ void OperatorEntry::updateDispatchTable_(const c10::Dispatcher& dispatcher, Disp
   auto dispatch_ix = static_cast<uint8_t>(dispatch_key);
   dispatchTable_[dispatch_ix] = computeDispatchTableEntry(dispatcher, dispatch_key);
   dispatchKeyExtractor_.setOperatorHasFallthroughForKey(dispatch_key, dispatchTable_[dispatch_ix].isFallthrough());
+  // Updates to alias dispatch key should trigger recomputation of all runtime keys mapped to the alias.
+  if (at::isAliasDispatchKey(dispatch_key)) {
+    for (auto k: c10::resolveAliasAutogradKey()) {
```

Comment:
Oh, I guess this is a case where having an iterator over the alias set is useful. I guess it's time to add iterator like functionality to dispatch key set...

Reply:
Yea created #43441 as a followup item for it :D